product_bom_revision/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 C2i Change 2 improve - Eduardo Magdalena <emagdalena@c2i.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product BOM revision",
+    "summary": "Store the revision of a product and a BOM",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "C2i Change 2 improve,"
+              "Odoo Community Association (OCA)",
+    "category": "Stock",
```

Comment:
@emagdalenaC2i Could you change this to 'Warehouse' as it won't appear in the good category in OCA shop?

Reply:
@rousseldenis ok, changed